server/sonar-web/src/main/js/apps/about/components/AboutApp.js

```diff
@@ -58,12 +58,14 @@ export default class AboutApp extends React.Component {
 
   loadData () {
     Promise.all([
-      window.sonarqube.appStarted,
       this.loadProjects(),
       this.loadIssues()
     ]).then(responses => {
       if (this.mounted) {
-        const [options, projectsCount, issues] = responses;
+        // FIXME
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)